src/Microsoft.TestPlatform.TestHostProvider/Hosting/DefaultTestHostManager.cs

```diff
@@ -469,8 +482,19 @@ private bool LaunchHost(TestProcessStartInfo testHostStartInfo, CancellationToke
             _processHelper.SetExitCallback(processId, ExitCallBack);
         }
 
+        if (_testHostProcess is null)
+        {
+            return false;
+        }
+
+        _testHostProcess.PriorityClass =
+            _environmentVariableHelper.GetEnvironmentVariable("VSTEST_HOST_PRIORITY") is { } processPriorityString
```

Comment:
Maybe we should add `INTERNAL` somewhere in the name to make it clear this is not stable API.